wkcuber/testdata/simple_wk_dataset/datasource-properties.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": {
+        "name": "simple_wk_dataset",
+        "team": ""
+    },
```

Comment:
I duplicated this dataset for now for a more simple setup, I thought that's fine since it's quite small.